recipes/AISHELL-1/ASR/transformer/hparams/train_ASR_transformer.yaml

```diff
@@ -0,0 +1,242 @@
+# ############################################################################
+# Model: E2E ASR with Transformer
+# Encoder: Transformer Encoder
+# Decoder: Transformer Decoder + (CTC/ATT joint) beamsearch
+# Tokens: BPE with unigram
+# losses: CTC + KLdiv (Label Smoothing loss)
+# Training: AISHELL-1
+# Authors:  Jianyuan Zhong, Titouan Parcollet
+# ############################################################################
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 8886
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+output_folder: !ref results/transformer/<seed>
+cer_file: !ref <output_folder>/cer.txt
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+# Data files
+data_folder: !PLACEHOLDER # e,g./path/to/aishell
+# noise/ris dataset will automatically be downloaded
+data_folder_rirs: !ref <data_folder> # Change this is needed
+skip_prep: False
+ckpt_interval_minutes: 15 # save checkpoint every N min
+train_data: !ref <output_folder>/train.csv
+valid_data: !ref <output_folder>/dev.csv
+test_data: !ref <output_folder>/test.csv
+tokenizer_file: https://drive.google.com/uc?export=download&id=1Fn1XJm0tlppGpgtZxs3Xzr7RuaxuRDLB
```

Comment:
@mravanelli not sure if we want to keep GDrive link. As Yannick mentioned, this is not a good long-term solution. It would be better to force peoples to retrain the tokenizer, no ?

Reply:
Yes, we will replace that with the HuggingFace one. Now I'm doing some decoding tests to see if I can replicate the results. Then, I can add the model to HuggingFace and change that.